templates/go/scaffolding/README.md

```diff
@@ -0,0 +1,10 @@
+# Go Scaffolding
+
+Organized into directories conforming to the naming convention
+[signature]-[invocation] where signature is the method signature group expected
```

Comment:
Also worth noting is that the scaffolding directories which are hidden (beginning with a dot ".") are not avialable when creating a new function, but are instead used for testing and for illustrative purposes.